core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -380,57 +363,24 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
-  def listGroupMembers(groupRef: RawlsGroupRef): Future[PerRequestMessage] = {
-    dataSource.inTransaction { dataAccess =>
-      dataAccess.rawlsGroupQuery.load(groupRef) flatMap {
-        case None => DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.NotFound, s"Group ${groupRef.groupName.value} does not exist")))
-        case Some(group) =>
-          dataAccess.rawlsGroupQuery.loadMemberEmails(group).map(memberEmails => RequestComplete(StatusCodes.OK, UserList(memberEmails)))
-      }
-    }
-  }
-
-  def createGroup(groupRef: RawlsGroupRef) = {
-    dataSource.inTransaction { dataAccess =>
-      dataAccess.rawlsGroupQuery.load(groupRef) flatMap {
-        case Some(_) => DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.Conflict, s"Group ${groupRef.groupName} already exists")))
-        case None =>
-          createGroupInternal(groupRef, dataAccess) map { _ => RequestComplete(StatusCodes.Created) }
-      }
-    }
-  }
-
-  private def createGroupInternal(groupRef: RawlsGroupRef, dataAccess: DataAccess): ReadWriteAction[RawlsGroup] = {
-    DBIO.from(gcsDAO.createGoogleGroup(groupRef)).flatMap { rawlsGroup =>
-      dataAccess.rawlsGroupQuery.save(rawlsGroup)
-    }
-  }
-
-  def deleteGroup(groupRef: RawlsGroupRef) = {
-    for {
-      group <- dataSource.inTransaction { dataAccess =>
-        withGroup(groupRef, dataAccess) { group =>
-          dataAccess.rawlsGroupQuery.delete(groupRef) map { _ => group }
-        }
-      }
-      _ <- gcsDAO.deleteGoogleGroup(group)
-    } yield {
-      RequestComplete(StatusCodes.OK)
-    }
-  }
-
   def overwriteGroupMembers(groupRef: RawlsGroupRef, memberList: RawlsGroupMemberList): Future[PerRequestMessage] = {
     for {
       (users, groups) <- dataSource.inTransaction({ dataAccess =>
         withGroup(groupRef, dataAccess) { group =>
           loadMemberUsersAndGroups(memberList, dataAccess)
         }
       }, TransactionIsolation.ReadCommitted) // read committed required to reduce db locks and allow concurrency
-
-      _ <- overwriteGroupMembership(groupRef, users.map(RawlsUser.toRef), groups.map(RawlsGroup.toRef))
+     _ <- overwriteGroupMembership(groupRef, users.map(RawlsUser.toRef), groups.map(RawlsGroup.toRef))
     } yield RequestComplete(StatusCodes.NoContent)
   }
 
+  /** completely overwrites all group members */
+  def overwriteGroupMembership(groupRef: RawlsGroupRef, users: Set[RawlsUserRef], subGroups: Set[RawlsGroupRef]): Future[RawlsGroup] = {
+    updateGroupMembershipInternal(groupRef) { group =>
+      group.copy(users = users, subGroups = subGroups)
+    }
+  }
```

Comment:
it's still used by `updateACL` and another function in `WorkspaceService.scala`

there's probably a way around that but i think it'd be way more surgery than it's worth right now since that code is being sam-ified in phase 4 anyway